src/main/charts/bitbucket/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: bitbucket
 description: A chart for installing Bitbucket Data Center on Kubernetes
 type: application
-version: 0.14.0
+version: 0.15.0
 appVersion: 7.14.1-jdk11
```

Comment:
The latest Bitbucket version is `7.15.1`, maybe we can upgrade it when we are doing changes anyway.
cc @wkritzinger-atlassian 
*note*: changing the version will update the `README.md` file for the chart and also require updated expected output test resource.